pctasks/run/pctasks/run/argo/client.py

```diff
@@ -130,33 +130,40 @@ def submit_workflow(
         executor_config: WorkflowExecutorConfig,
         runner_image: str,
     ) -> Dict[str, Any]:
+        print("=== SUBMITTING WORKFLOW ===")
```

Comment:
OK to remove all these prints?

Reply:
Yeah, all the `prints` were just to help in my remote print statement debugging, feel free to delete any/all.